packages/dynatrace-oneagent-operator/generated-changes/overlay/app-readme.md

```diff
@@ -1,6 +0,0 @@
-# Dynatrace OneAgent Operator
```

Comment:
`dynatrace-oneagent-operator` had changes stored in the patch file as well as overlaid on top, so this change was mostly the scripts adjusting to storing the minimum required diff / overlay (e.g. just a patch on the questions.yml since the upstream app-readme.md is equivalent to this).